Cargo.toml

```diff
@@ -2,41 +2,42 @@
 name = "mongo-task-generator"
 description = "Dynamically split evergreen tasks into subtasks for testing the mongodb/mongo project."
 license = "Apache-2.0"
-version = "0.7.15"
+version = "0.7.16"
 repository = "https://github.com/mongodb/mongo-task-generator"
 authors = ["Decision Automation Group <dev-prod-dag@mongodb.com>"]
 edition = "2018"
 
 [dependencies]
-anyhow = "1.0"
-async-trait = "0.1"
-chrono = "0.4"
-clap = "3.1"
-cmd_lib = "1.3"
-futures = "0.3"
-lazy_static = "1.4"
-maplit = "1"
-rand = "0.8.5"
-regex = "1"
-reqwest = "0.11.13"
-reqwest-middleware = "0.2.0"
-reqwest-retry = "0.2.0"
-serde = { version = "1.0", features = ["derive"] }
-serde_json = "1.0"
-serde_yaml = "0.8"
-shellexpand = "2.1"
-shrub-rs = "0.5.2"
-tokio = { version = "1", features = ["full"] }
-tracing = "0.1"
-tracing-subscriber = { version = "0.3", features = ["json", "fmt", "std"] }
+anyhow = "^1.0.86"
```

Comment:
https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#caret-requirements

"Leaving off the caret is a simplified equivalent syntax to using caret requirements. While caret requirements are the default, it is recommended to use the simplified syntax when possible."